pkg/features/features.go

```diff
@@ -86,6 +86,11 @@ const (
 	// owner: @docktofuture
 	// beta: v1.133.0
 	CustomDNSServerInNodeLocalDNS featuregate.Feature = "CustomDNSServerInNodeLocalDNS"
+
+	// VpnBondingModeRoundRobin enables the usage of the "balance-rr" bonding mode for the HA VPN setup.
+	// owner: @domdom82
+	// beta: v1.133.0
+	VpnBondingModeRoundRobin featuregate.Feature = "VpnBondingModeRoundRobin"
```

Comment:
You also need to update `docs/deployment/feature_gates.md`.

Reply:
done